backend/src/app.js

```diff
@@ -16,8 +16,8 @@ app.use(
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-app.use('/api/v1/users', UserRoutes);
+app.use('/api/auth', UserRoutes);
 
-app.use(globalErrorHandler);
+// app.use(globalErrorHandler);
```

Comment:
Do you plan to use this later on? if not, can we get rid of this un-necessary comment?

Reply:
I commented it out earlier due to an unintuitive error in the terminal that caused difficulties. Later, Hena Apu updated the global error handler, but I forgot to uncomment it. I've now resolved the issue. Thank you.